tests/test_x509_ext.py

```diff
@@ -3003,6 +3004,20 @@ def test_distribution_point_full_and_relative_not_none(self):
         with pytest.raises(ValueError):
             x509.DistributionPoint("data", "notname", None, None)
 
+    def test_relative_name_name_value_deprecated(self):
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
pytest has a function for this `pytest.deprecated_call()` which can be called as a contextmanager.